### PR TITLE
[DEV-1261] Add quickstart guide strapi side

### PR DESCRIPTION
**Pull request by tommaso1**
#### List of Changes

Add quickstart guides and quickstart guide items strapi side

#### Screenshots (if appropriate):

QuickstartGuide:
image
image

QuickstartGuideItem:
image
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/strapi-cms/src/api/tutorial/content-types/tutorial/schema.json
Comment (MarcoPonchia) [inline apps/strapi-cms/src/api/tutorial/content-types/tutorial/schema.json:1]: i think this file belong to another PR

File: apps/strapi-cms/src/api/tutorial/controllers/tutorial.ts
Comment (MarcoPonchia) [inline apps/strapi-cms/src/api/tutorial/controllers/tutorial.ts:1]: same here

File: apps/strapi-cms/src/api/tutorial/routes/tutorial.ts
Comment (MarcoPonchia) [inline apps/strapi-cms/src/api/tutorial/routes/tutorial.ts:1]: same here

File: apps/strapi-cms/src/api/tutorial/services/tutorial.ts
Comment (MarcoPonchia) [inline apps/strapi-cms/src/api/tutorial/services/tutorial.ts:1]: same here

File: apps/strapi-cms/src/api/quickstart-guide-item/content-types/quickstart-guide-item/schema.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "type": "string"
+    },
+    "content": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "blocks"
+    }
```
Comment (MarcoPonchia) [inline apps/strapi-cms/src/api/quickstart-guide-item/content-types/quickstart-guide-item/schema.json:47]: Some attribute are missing and i think `content` should be more complex in NextJS `QuickStartGuide` type manage multiple steps `apps/nextjs-website/src/lib/types/step.ts` and in each step can show different types of bloc `apps/nextjs-website/src/lib/types/part.ts`

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**